includes/discovery/sensors/state/sitemonitor.inc.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * sitemonitor.inc.php
+ *
+ * LibreNMS state discovery module for Packetflux SiteMonitor Base II
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package    LibreNMS
+ * @link       http://librenms.org
+ * @copyright  2020 Josh Baird
+ * @author     Josh Baird <joshbaird@gmail.com>
+ */
+
+$switch = snmp_get($device, ".1.3.6.1.4.1.32050.2.1.26.5.3", "-Ovqe");
```

Comment:
I was told I cannot do this in YAML since there is no MIB for this specific device (Packetflux does not provide one).  See https://community.librenms.org/t/adding-state-sensor/10521/5 for more details.